src/Lemon/Containers/Converters/NodeContainer2BinaryCia.cs

```diff
@@ -116,6 +131,77 @@ void WriteFile(DataWriter writer, Node file, bool isRequired = true)
             writer.WritePadding(0x00, BlockSize);
         }
 
+        void UpdateTitleMetadata(TitleMetadata title, Node content)
+        {
+            int chunksHashed = 0;
+            var infoRecords = new DataStream();
```

Comment:
Add `using`, so the Stream is disposed/closed once the method finishes. Like `using var infoRecords = new DataStream();`